applications/RomApplication/tests/test_calculate_rom_basis_output_process.py

```diff
@@ -35,14 +34,36 @@ def testCalculateRomBasisOutputProcess(self):
             "nodal_unknowns": ["TEMPERATURE"],
             "rom_basis_output_format": "json",
             "rom_basis_output_name": "RomParameters",
-            "svd_truncation_tolerance": 1.0e-6
+            "svd_truncation_tolerance": 1.0e-6,
+            "append_hrom_settings": false
         }""")
 
         # Run a "fake" simulation to calculate the ROM basis from its results
         self.__ExecuteTest(self.process_settings)
 
         # Check results
-        self.__CheckResults()
+        check_hrom_settings = False
+        self.__CheckResults(check_hrom_settings)
+
+    def testCalculateRomBasisOutputProcessHRomSettings(self):
+        # Create a CalculateROMBasisOutputProcess instance
+        self.process_settings = KratosMultiphysics.Parameters("""{
+            "model_part_name": "MainModelPart",
+            "snapshots_control_type": "step",
+            "snapshots_interval": 1.0,
+            "nodal_unknowns": ["TEMPERATURE"],
+            "rom_basis_output_format": "json",
+            "rom_basis_output_name": "RomParametersWithHRom",
+            "svd_truncation_tolerance": 1.0e-6,
+            "append_hrom_settings": true
```

Comment:
I thought the `append_hrom_settings` was to be defalted to `false`

Reply:
I changed my mind while doing the PR. In any case, I'd follow your suggestion to always write these files.

Reply:
Done.